pipelines/measurement/measurement_matched.yaml

```diff
@@ -8,16 +8,6 @@ tasks:
       python: |
         from lsst.faro.measurement import PA1Task
         config.measure.retarget(PA1Task)
-  PA2_design:
-    class: lsst.faro.measurement.TractMatchedMeasurementTask
-    config:
-      connections.package: validate_drp
-      connections.metric: PA2_design_gri
-      python: |
-        from lsst.faro.measurement import PA2Task
-        config.measure.retarget(PA2Task)
```

Comment:
I know this is being removed, I just wanted to comment for future info. A python block is always evaluated prior to the key value pairs, so it is safe to do the `config.measure.retarget(PA2Task)` and have `config.measure.threshPA2: 15.0` in the key value list. Maybe this would be more clear if we changed the order in which pipelines are printed. It is absolutely ok to have it as written as well though.

Reply:
Thanks Nate, we have many other pipelines so I'll create another ticket to update them all along the lines you suggest. This ticket is just fixing documentation build issues.